src/e2e/__tests__/caret.ts

```diff
@@ -39,7 +36,14 @@ describe('caret testing', () => {
 
     await page.keyboard.press('Enter')
     await paste(page, [''], importText)
-    await setCursor(page, ['puppeteer', 'web scrapping'], { offset: 3 })
+
+    await clickBullet(page, 'puppeteer')
+    await clickBullet(page, 'web scrapping')
```

Comment:
Note that `setCursor(page, ['puppeteer', 'web scrapping'])` sets the cursor once (on "web scrapping" [sic]). It just takes the full `Context` as an argument. So you should replace this with a single click.

Reply:
I couldn't understand what you mean. How can I click the collapsed subthought with a single click?

Reply:
Oops! You’re right :).